assets/app/js/controllers/conclusionDialog.js

```diff
@@ -221,7 +226,12 @@ class ConclusionDialog extends Controller {
           data: $.unique(copyright_statements),
           multiple: true,
           placeholder: 'Enter copyright',
-          tags: true
+          tags: true,
+          language: {
+            'noResults': function () {
+              return (`SCWB did not find any copyright statements in the scan pertaining to ${conclusion.path}`);
```

Comment:
In some cases, this `conclusion.path` value is very long and is awkward to show in a small menu item. 

Instead of `${conclusion.path}`, we should have "the current path".